feat: expose rowHeight from onHeightChange (#284)
Andarist/react-textarea-autosize: .changeset/fair-bobcats-matter.md

> ---

Reviewer: Could you also update the README?

Author: The description of the `onHeightChange` in the table?

Reviewer: Ye

'react-textarea-autosize': minor
---

> Expose `{ rowHeight: number }` as second argument of the callback function `onHeightChange`. This is useful to construct custom behaviors according to the height values.

Reviewer: ```suggestion
Added `{ rowHeight: number }` as a second parameter to the `onHeightChange` callback. This is useful to construct custom behaviors according to the height values.
```
